admin/acceptance.py

```diff
@@ -19,6 +20,33 @@
 import flocker
 
 
+def safe_call(command, **kwargs):
+    """
+    Run a process and kill it if the process is interrupted.
+
+    Takes the same arguments as ``subprocess.Popen``.
+    """
+    process = Popen(command, **kwargs)
+    try:
+        return process.wait()
+    except:
```

Comment:
What's expected here?  `SystemExit`?  Something wrapping `EINTR`?  A random oops-this-code-isn't-signal-safe exception from the `Popen` implementation?  And why doesn't this exception get re-raised after `process.terminate()`?